assets/src/application/ui/display/espressoIcons/EspressoIcon.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { svgPath } from './svgPaths';
+import { EspressoIconProps } from './types';
+
+/**
+ * Custom Event Espresso Dashicons
+ */
+const EspressoIcon = React.forwardRef<SVGSVGElement, EspressoIconProps>(
+	({ name, svgSize = 20, className, ...otherProps }, ref) => {
+		const path = svgPath[name];
+
+		if (!path) {
+			return null;
+		}
+
+		const htmlClass = classNames(className, 'dashicon', `dashicons-${name}`, 'espresso-icon');
+
+		return (
+			<svg
+				role={'img'}
```

Comment:
why not `role='img'`?
want to understand the difference 
between this :point_up: and :point_right: `role={'img'}`

Reply:
there's no difference technically

I usually use brackets for string props because then it is easier to convert the value to a variable (not the case for the "role" attribute of course) and i can just do the same thing every time out of habit without having to think about it.

just my two cents worth, but if both of you really prefer using the `prop='string'` pattern over  `prop={'string'}` then i am happy to change.

Reply:
I am ok with both but would prefer a consistent approach, such as there is no need to think twice when seeing smth basic like this